docs/setup/independent/high-availability.md

```diff
@@ -282,7 +282,7 @@ Please select one of the tabs to see installation instructions for the respectiv
    EOF
    ```
 
-   Make sure you replace `<etcd0-ip-address>`, `<etcd1-ip-address>` and `<etcd2-ip-address>` with the appropriate IPv4 addresses.
+   Make sure you replace `<etcd0-ip-address>`, `<etcd1-ip-address>` and `<etcd2-ip-address>` with the appropriate IPv4 addresses. Also, make sure that you changed `<etcd0>`, `<etcd1>` and `<etcd2>` with real hostnames of each machine and that these machines are able to reach each other using DNS or add records to `/etc/hosts`.
```

Comment:
| add records to `/etc/hosts`.

records added to `/etc/hosts`.

Reply:
Nit.."Make sure you replace" vs "make sure that you changed" ... it would be nice to have both sentences use consistency. Thanks!

Reply:
I hope it's good now.